### ISSUE TITLE
Installing gems / puppet-lint may break puppet-enterprise

### ISSUE THREAD

**Issue by fullmooninu** [rodjek/puppet-lint#358]
I haven't thouroughly tested this out, but it's the second time I'm seeing it.

If you install puppet-lint on your ***puppet-enterprise*** machine, or try to install ***gem*** so you can install/use the latest version of ***puppet-lint***, you'll ruin your puppet-enterprise master machine.

Reason is, I think, puppet enterprise is using it's own gem and ruby environment, and installing gem and ruby stuff will break that.

Since puppet enterprise uses an old version of puppet, I'm not sure if this is valid on normal, more recent puppet installations.

So, to avoid all this mess, and since it's really useful (for me) to have a running puppet master installation where i can quickly and repeatedly edit  .pp-> ```puppet agent -t```  stuff, can you consider the suggestion of releasing a binary of puppet-lint?

**Comment by rodjek** [comment]
I haven't used puppet enterprise, so I'm not sure of its particulars but if it vendors its own ruby install, can you install puppet-lint with their version of `gem` rather than the one provided by your OS (`/opt/puppet/bin/gem install puppet-lint`)?

**Comment by rnelson0** [comment]
@fullmooninu You should be able to install the gem with the correct vendored binaries. If you have issues, you should open a Puppet support ticket, who can help determine the cause and the fix. If there is a process that would help, please open an issue/PR with the information required to update the README. Thanks!
